virtcontainers/kata_agent.go

```diff
@@ -34,6 +34,7 @@ import (
 )
 
 var (
+	checkRequestTimeout   = 10 * time.Second
```

Comment:
You can bump this to 30s IMO, this won't hurt, and will cover for extreme cases.